elixir/mix.exs

```diff
@@ -3,7 +3,7 @@ defmodule RabbitmqTutorials.Mixfile do
 
   def project do
     [app: :rabbitmq_tutorials,
-     version: "0.0.1",
+     version: "0.0.2",
```

Comment:
While it doesn't matter for this specific "project", I cannot accept a PR that messes with the version number. It's up to the maintainers to decide when to bump it and why.

Reply:
That's cool. I was merely trying to distinguish it from the previous version that used the pre-1.0 version of the `amqp` library.